docs/modules/ROOT/pages/documentation/payara-server/rest-api/rest-api-documentation.adoc

```diff
@@ -7,6 +7,488 @@ documentation is based on the
 https://github.com/OAI/OpenAPI-Specification[OpenAPI specification]
 (formerly known as Swagger v2.0).
 
-The documentation presented here has been automatically generated using
-the https://github.com/Swagger2Markup/swagger2markup[Swagger2Markup] (ASCIIDoc support included)
-extension.
+[[payara-server-administration-and-monitoring-api]]
+== Payara Server Administration and Monitoring API
+
```

Comment:
Try to avoid having no text between two headings - it implies a formatting error to the user. Have a look here: https://payara.atlassian.net/wiki/spaces/SUP/pages/2149515318/Payara+Documentation+Style+Guide#Headings.

You can remove the `Payara Server Administration and Monitoring API` heading as it is a bit redundant, or include some text here